app/v4/editor/enable-betas.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Enable DIVA Client beta
+category: editor
+topic: client
+sortOrder: 1000
+---
+
+We may occasionally ask you to enable betas on a specific computer. 
+We don't recommend using this without contacting us first.
+
+## Enable DIVA Client beta:
+
+1. Sign into DIVA Client.
+
+1. Click your **username** in the top right corner.
+
+1. Click **Preferences**.
+
+1. Under **Application**, click the **Enable betas** toggle.
+
+<p class="tip tip--result">
+  <strong>Result</strong>:
+  Betas are now enabled. If there is one avaliable, it will automatically download and install next time you restart DIVA Client. 
+  Alternatively you can force the install by clicking on the the parachute icon in the top right corner and follow the on screen instructions.
```

Comment:
Can you please add the font awesome parachute icon here?

Reply:
you need to update font awesome